app/controllers/games_controller.rb

```diff
@@ -1,3 +1,7 @@
 class GamesController < ApplicationController
   def index; end
+
+  def show
```

Comment:
The Rubocop error wants you to write this method like the `index` method since it's empty:

```ruby
def show; end
```